README.md

```diff
@@ -97,29 +97,33 @@ In this GitHub repository, plugins that address the same feature (cache, lazyloa
 
 It’s easy to over-engineer naming conventions, so we stick to a pragmatic rule of thumb when in doubt:
 
-1. Namespaces should be unique and consistent.
+1. Namespaces should be unique and consistent, and written in CamelCase.
 2. Function names should be descriptive.
 
 We rely on one consistent namespace for all Helper Plugins, and descriptive subnamespaces for each plugin:
 
 ```php
-namespace ImagifyPlugin\Helpers\{subnamespace(s)}
+namespace ImagifyPluginHelpers\{subnamespace(s)}
```

Comment:
It's better to start with a vendor name first and then the package name:

```php
namespace WPMedia\ImagifyPluginHelpers;
```

The subnamespace(s) are optional and would follow the developer or dev team's coding standards.

Reply:
The subnamespace(s) are not optional here, each helper must have its own unique namespace.